CONTRIBUTING.md

```diff
@@ -147,57 +150,78 @@ def make_squares(key, value=0):
+* `joined_lower` or ALL_CAPS for constants.
+* `StudlyCaps` for classes.
+* `camelCase` only to conform to pre-existing conventions.
+* Attributes: `interface`, `_internal`, `__private`
+* Try to avoid the `__private` form.
+
+## Testing and code coverage
+
+The application should be unit tested (at least a minimal set of unit tests should be implemented in the `/test` directory). We should be able to measure the unit test code coverage.
+
+Run a unit test suite to make sure your changes do not break current implementation:
+
 ```
 $ cd <package>
 $ python setup.py test
 ```
-# Code coverage
-To measure application code coverage for unit tests please use the coverage tool. This set of commands will locally create a code coverage report for all unit tests:
+
+## Code coverage
+
+To measure application code coverage for unit tests, please use the coverage tool. This set of commands locally creates a code coverage report for all unit tests:
+
 ```
 $ cd <package>
 $ coverage run setup.py test
 $ coverage report
 $ coverage html
 ```
-Last command will generate for you useful HTML code coverage report. It can be used to check which parts of your code are not unit tested.
 
-# Keep your GitHub fork updated
-I want to fork a GitHub repo SOME_REPO/appname to USER/appname and want to keep it updated.
-### Tracking changes
+The last command generates an HTML code coverage report. You can use it to check which parts of your code are not unit tested.
+
+## Keep your GitHub fork updated
+
+To fork a GitHub repo SOME_REPO/appname to USER/appname and keep it updated, please use the following steps.
+
+### Track changes
+
 ```
 $ git clone git@github.com:USER/appname.git
 $ cd appname
 $ git remote add upstream git@github.com:SOME_REPO/appname.git
 ```
-### Verify with:
+
+### Verify:
+
+Verify with the following:
+
 ```
 $ git remote -v
 origin  https://github.com/USER/appname.git (fetch)
 origin  https://github.com/USER/appname.git (push)
 upstream  https://github.com/SOME_REPO/appname.git (fetch)
 upstream  https://github.com/SOME_REPO/appname.git (push)
 ```
-### Updating
-Each time I want to update, from my local master branch I will do the following:
+
+### Update
+
+To update, do the following from the local master branch:
+
 ```
 $ git fetch upstream
 $ git rebase upstream/master
 ```
-The goal of the rebase is to have a cleaner history if I have local changes or commits on the repo.
-# Final notes
-1. Please do not change the version of the package in the ``setup.py`` file. The person or process responsible for releasing will do this and release the new version.
-2. Keep your GitHub updated! Please make sure you are rebasing your local branch with changes so you are up to date and it is possible to automatically merge your pull request.
-3. Please, if possible, squash your commits before pushing to the remote repository.
+
+Rebasing creates a cleaner history for those with local changes or commits on the repo.
```

Comment:
I'd recommend changing this line to `Rebasing creates a cleaner history for developers with local changes or commits.`